### PR TITLE
Critical last second pre-freeze PR

### PR DESCRIPTION
**Pull request by Bawhoppen**

### CODE REVIEW

**Comment by RemieRichards** [comment]
It's not pre-freeze, we go by Australia rules.

**Comment by Bawhoppen** [comment]
FUCK

**Comment by Bawhoppen** [comment]
merge anyways yolo

File: code/modules/hydroponics/grown.dm [hunk 1]
```diff
@@ -601,6 +601,10 @@
 	trash = /obj/item/weapon/grown/bananapeel
 	filling_color = "#FFFF00"
 
+/obj/item/weapon/reagent_containers/food/snacks/grown/banana/suicide_act(mob/user)
+		user.visible_message("<span class='suicide'>[user] horrifically mutilates \himself with the [src.name] in a way you cannot even comprehend! It looks like \he's trying to commit suicide.</span>")
```
Comment (RemieRichards) [inline code/modules/hydroponics/grown.dm]: these are too far indented.

**Comment by Supermichael777** [comment]
From what I'm told the server is in Miami so its ether that or MSO's house.
Also what Australia, honk?

**Comment by RemieRichards** [comment]
@Supermichael777 freezes have always been held with Australia rules.

**Comment by ike709** [comment]
this isn't a feature, this is a critical bugfix

**Comment by Cheridan** [comment]
reopen after the freeze

**Comment by Bawhoppen** [comment]
reopen please

**Comment by Bawhoppen** [comment]
im retarded

**Pull request closed without merging**